### PR TITLE
Birth date fix

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/components/Graph/Navigation/Navigation.scss [hunk 1]
```diff
@@ -38,12 +38,11 @@
...
       svg {
-        margin: 0.5em;
         font-size: 0.65em;
-        margin-right: 10px;
+        margin: 0.5em 10px 0.5em 0.5em;
       }
-      padding: 0;
-      padding-right: 0.5em;
+
+      padding: 0 0.5em 0 0;
```
Comment (ogroppo) [inline src/components/Graph/Navigation/Navigation.scss:45]: how come this choice? possible to have a screenshot?

File: src/lib/addEntityConnectors.js [hunk 2]
```diff
@@ -26,16 +27,25 @@ export default function addEntityConnectors(entity, propId, options = {}) {
...
   if (options.addLeftIds) entity.leftIds = getClaimIds(entity, SIBLINGS_ID);
   else delete entity.leftIds;
 }
 
-function addRightIds(entity) {
-  const claim = entity.claims[SPOUSE_ID] || []; //cannot use simpleclaims here as only preferred will show up
+function addRightIds(entity, options) {
+  let claim = [];
+  console.log(options.rightEntityType);
+  if (options.rightEntityType === "Spouses and partners") {
```
Comment (ogroppo) [inline src/lib/addEntityConnectors.js:41]: are you not using the codes for this?

File: src/lib/formatEntity.js [hunk 3]
```diff
@@ -108,7 +109,7 @@ export default async function formatEntity(
 function addIsHuman(entity) {
   try {
     entity.isHuman = entity.simpleClaims[INSTANCE_OF_ID].some(
-      ({ value }) => value === HUMAN_ID //add all other IDS of person subclass?
+      ({ value }) => value === HUMAN_ID || value === FICTIONAL_HUMAN_ID
```
Comment (ogroppo) [inline src/lib/formatEntity.js:112]: ❤️

File: src/lib/sortEntities.js [hunk 4]
```diff
@@ -1,16 +1,22 @@
 import { BIRTH_DATE_ID, GENDER_ID } from "../constants/properties";
 import getDateNumber from "./getDateNumber";
 import { ANIMAL_MALE_ID, HUMAN_MALE_ID } from "../constants/entities";
+
+/*
+Sort entities by birth date, youngest child will be on the left, if birth date is unknown they will be on the left side
```
Comment (ogroppo) [inline src/lib/sortEntities.js:6]: very useful comment

**Review by ogroppo — APPROVED** [review]
Feel free to merge if you have tested it

File: src/components/Graph/Navigation/Navigation.scss (hunk 1 above)
Comment (mshd) [reply src/components/Graph/Navigation/Navigation.scss:45]: It's just another way of writing css in one-line, it doesnt change anything

File: src/lib/addEntityConnectors.js (hunk 2 above)
Comment (mshd) [reply src/lib/addEntityConnectors.js:41]: Not currently

File: src/lib/sortEntities.js (hunk 4 above)
Comment (mshd) [reply src/lib/sortEntities.js:6]: Thank you!

**Pull request merged**